config/config.go

```diff
@@ -1,45 +1,51 @@
 package config
 
-import "github.com/kelseyhightower/envconfig"
+import (
+	"context"
+	"time"
+
+	"github.com/kelseyhightower/envconfig"
+)
 
 // Config represents service configuration for dp-frontend-filter-dataset-controller
 type Config struct {
-	BindAddr             string `envconfig:"BIND_ADDR"`
-	RendererURL          string `envconfig:"RENDERER_URL"`
-	FilterAPIURL         string `envconfig:"FILTER_API_URL"`
-	DatasetAPIURL        string `envconfig:"DATASET_API_URL"`
-	HierarchyAPIURL      string `envconfig:"HIERARCHY_API_URL"`
-	DatasetAPIAuthToken  string `envconfig:"DATASET_API_AUTH_TOKEN"`
-	FilterAPIAuthToken   string `envconfig:"FILTER_API_AUTH_TOKEN"`
-	SearchAPIAuthToken   string `envconfig:"SEARCH_API_AUTH_TOKEN"`
-	SearchAPIURL         string `envconfig:"SEARCH_API_URL"`
-	DownloadServiceURL   string `envconfig:"DOWNLOAD_SERVICE_URL"`
-	EnableDatasetPreview bool   `envconfig:"ENABLE_DATASET_PREVIEW"`
-	EnableLoop11         bool   `envconfig:"ENABLE_LOOP11"`
+	BindAddr                   string        `envconfig:"BIND_ADDR"`
+	RendererURL                string        `envconfig:"RENDERER_URL"`
+	FilterAPIURL               string        `envconfig:"FILTER_API_URL"`
+	DatasetAPIURL              string        `envconfig:"DATASET_API_URL"`
+	HierarchyAPIURL            string        `envconfig:"HIERARCHY_API_URL"`
+	DatasetAPIAuthToken        string        `envconfig:"DATASET_API_AUTH_TOKEN" json:"-"`
+	FilterAPIAuthToken         string        `envconfig:"FILTER_API_AUTH_TOKEN"  json:"-"`
+	SearchAPIAuthToken         string        `envconfig:"SEARCH_API_AUTH_TOKEN"  json:"-"`
+	SearchAPIURL               string        `envconfig:"SEARCH_API_URL"`
+	DownloadServiceURL         string        `envconfig:"DOWNLOAD_SERVICE_URL"`
+	EnableDatasetPreview       bool          `envconfig:"ENABLE_DATASET_PREVIEW"`
+	EnableLoop11               bool          `envconfig:"ENABLE_LOOP11"`
+	HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
+	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
 }
 
 var cfg *Config
 
 // Get returns the default config with any modifications through environment
 // variables
-func Get() (*Config, error) {
-	if cfg != nil {
-		return cfg, nil
-	}
+func Get(ctx context.Context) (cfg *Config, err error) {
```

Comment:
do we need ctx?

Reply:
No we don't, it was added in as I was moving away from the previous config.go file and then forgot to remove it